### PR TITLE
FuzzyLocal

### PR DESCRIPTION
**Pull request by SilverBrother**
Levensthein simplificeret (ren levensthein og ikke effektiv Fastenshtein)

### CODE REVIEW

**Comment by Murmeldyret** [comment]
For fuzzy matching the program should use the library instead

**Comment by SilverBrother** [comment]
Fastenshtein library hentet ned (original algoritme udkommenteret) og klar til brug.
Ngramiser også klar.
pga. 4 fejl kan jeg ikke lave unit test af dem. Se fejl forneden:

- Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'Reference' could not be found (are you missing a using directive or an assembly reference?)	zenref	C:\Users\Erik_\Desktop\AAU\3. Semester\Projekt\P3-Project\zenref\src\Models\Api.cs	27	Active

- Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'Reference' could not be found (are you missing a using directive or an assembly reference?)	zenref	C:\Users\Erik_\Desktop\AAU\3. Semester\Projekt\P3-Project\zenref\src\Models\Api.cs	27	Active

- Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0234	The type or namespace name 'Models' does not exist in the namespace 'zenref' (are you missing an assembly reference?)	zenref.Tests	C:\Users\Erik_\Desktop\AAU\3. Semester\Projekt\P3-Project\zenref.Tests\UnitTest1.cs	3	Active

- Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0029	Cannot implicitly convert type 'zenref.Models.Reference' to 'bool'	zenref.Tests	C:\Users\Erik_\Desktop\AAU\3. Semester\Projekt\P3-Project\zenref.Tests\SpreadsheetTest.cs	36	Active

File: zenref.Tests/ReferenceTest.cs [hunk 1]
```diff
@@ -14,10 +14,10 @@ public void Fuzzy_search_locally()
             string testValue = "hello2";
 
             //Act
-            int result = Reference.FuzzyLocal(testString, testValue);
+            int result = Reference.Fuzzy(testString, testValue);
 
             //Assert
-            Assert.True(result > 75);
+            Assert.True(result < 3);
```
Comment (Murmeldyret) [inline zenref.Tests/ReferenceTest.cs]: Kan man ikke give et mere konkret svar på, hvad result er?

File: zenref.Tests/ReferenceTest.cs [hunk 2]
```diff
@@ -28,10 +28,10 @@ public void Fuzzy_search_online()
             string testValue = "hello2";
 
             //Act
-            int result = Reference.FuzzyOnline(testString, testValue);
+            int result = Reference.Fuzzy(testString, testValue);
 
             //Assert
-            Assert.True(result > 75);
+            Assert.True(result < 3);
```
Comment (Murmeldyret) [inline zenref.Tests/ReferenceTest.cs]: Samme som linje 20

**Review by Murmeldyret — COMMENTED** [review]
The tests could be improved to assert for a more precise number.

**Review by Murmeldyret — COMMENTED** [review]
Better

**Review by Murmeldyret — APPROVED** [review]

**Pull request merged**